axis.go

```diff
@@ -5,14 +5,17 @@
 package plot
 
 import (
-	"fmt"
 	"image/color"
 	"math"
+	"strconv"
 
 	"github.com/gonum/plot/vg"
 	"github.com/gonum/plot/vg/draw"
 )
 
+// sane is a sane level of float precision for a plot.
+const sane = 4
```

Comment:
The word or the value? I'd say a precision of anything more than 1 part in 10,000 on a plot is probably more than necessary, but it is an upper bound on the precision.